
def local_errors():
    pass